zebra/zapi_msg.h

```diff
@@ -84,13 +84,9 @@ extern int zsend_route_notify_owner_ctx(const struct zebra_dplane_ctx *ctx,
 
 extern void zsend_rule_notify_owner(const struct zebra_dplane_ctx *ctx,
 				    enum zapi_rule_notify_owner note);
-extern void zsend_ipset_notify_owner(struct zebra_pbr_ipset *ipset,
-				     enum zapi_ipset_notify_owner note);
-extern void
-zsend_ipset_entry_notify_owner(struct zebra_pbr_ipset_entry *ipset,
-			       enum zapi_ipset_entry_notify_owner note);
-extern void zsend_iptable_notify_owner(struct zebra_pbr_iptable *iptable,
-				       enum zapi_iptable_notify_owner note);
+
+extern void zsend_netfilter_notify_owner(const struct zebra_dplane_ctx *ctx,
```

Comment:
can 'ctx' be 'const' here?

Reply:
I think yes, like for zsend_rule_notify_owner.